src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -43,7 +43,7 @@
         "symfony/uid": "^6.4|^7.0",
         "symfony/validator": "^6.4|^7.0",
         "symfony/var-dumper": "^6.4|^7.0",
-        "doctrine/collections": "^1.0|^2.0",
+        "doctrine/collections": "^1.8|^2.0",
```

Comment:
We need to add a conflict rule to prevent an older version of the package to be installed.